go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/prometheus/alertmanager
 
-go 1.23.0
+go 1.24
 
 toolchain go1.24.1
```

Comment:
NIT: I really dislike go polluting `go.mod` with toolchain so eagerly, AFAIK they fixed it in master. We could clean this up by having `go 1.24.0` and removing `toolchain` line.